docs/_includes/api/create_database.html

```diff
@@ -31,7 +31,9 @@
 
 **WebSQL-only options:**
 
-* `size`: Amount in MB to request for storage, which you will need if you are storing >5MB in order to [avoid storage limit errors on iOS/Safari](/errors.html#not_enough_space). Persistent views use a separate database per view which will also request storage space so keep in mind the upper storage limits when using persistent views on iOS.
+_Note:_ as of PouchDB 7.0.0, this requires installing WebSQL as an add-on adapter.
```

Comment:
I think this and the other WebSQL note just below should probably move this to the WebSQL section